Cloud Application Manager/Core Concepts/boxes.md

```diff
@@ -37,17 +38,41 @@ Boxes contain scripts, variables, and metadata to automate processes when instan
 
-### Understanding Box Basics
+### Boxes Page
 
-#### New Box
+When you click on the **Boxes** left side menu the boxes page appears displaying all your available boxes in the current scope. You can display them as **tiles** with some information for each box, or as a **list** with a few columns showing basic information for them. To switch between these views you can click a button on the top-right corner of the page.
 
-To create a new box, click **New**. Select a box type to match your automation:
+![Box page in tiles view](../../images/cloud-application-manager/core-concepts/boxes-tiles.png)
 
-* [Application](../Automating Deployments/application-box.md) - Configure several boxes to deploy an application with a single click.
-* [Script](../Automating Deployments/script-box.md) - Automate using Bash, PowerShell, Salt, Ansible, Puppet, or Chef.
-* [Container](../Automating Deployments/docker-container-service.md) - Automate using container technology like Docker.
-* [Template](../Automating Deployments/template-box.md) - Automate using AWS CloudFormation templates, ARM templates or Terraform templates.
-* [Deployment Policy](../Automating Deployments/deploymentpolicy-box.md) - Select and share infrastructure resources, networking, and more from a cloud provider.
+The **tiles view** displays a big icon, name, type and the owner of the box. When you hover on a tile, two action buttons appear allowing you to deploy or manage the box. You can sort the tiles by name (default), last modified or owner by selecting one of these values in the *Sort by* dropdown located on the top-right corner of the page, next to the change view mode button.
+
+![Box page in list view](../../images/cloud-application-manager/core-concepts/boxes-list.png)
+
+The **list view** displays for each element a small icon, name, type, when it was last modified, the owner and two action buttons for each box. You can order by any column by clickin on its header and f you select any of the boxes in the list, a *Bulk actions* button appears, so that selected boxes can be updated all at once.
+
+The *gear icon* action button allows you to manage the box by performing additional actions such as sharing the box, editing its details, cloning or deleting it. These options will be available depending on your access permissions onto the box.
+
+You can also use the *filters* above to narrow down the boxes being displayed by the workspace they belong to or the tags they have associated.
+
+There are several types of boxes to address for your specific automation use case:
+
+* [Application boxes](../Automating Deployments/application-box.md) allows to configure several boxes to deploy an application with a single click.
```

Comment:
I doubt here: should we use singular (regarding a type) or plural (regarding boxes)? In both cases some verbs (don't) need an 's'